Refresh check-spelling 0.0.26 (#47119)
microsoft/PowerToys: doc/devdocs/modules/fancyzones.md

FancyZones cpp project read/write config data handler is in FancyZonesLib project.

![Debug Step Image](../images/fancyzones/19.png)

> However, the files read and write those are C:\Users\“xxxxxx”\AppData\Local\Microsoft\PowerToys\FancyZones

Author: This "sentence" doesn't really make sense. But I can't fix it since I don't understand it.

The read+write is an example of an irreversible binomial

> However, the files read and write those are C:\Users\“xxxxxx”\AppData\Local\Microsoft\PowerToys\FancyZones

Reviewer: ```suggestion
However, the files read from and written to are those in `C:\Users\“xxxxxx”\AppData\Local\Microsoft\PowerToys\FancyZones`
```

Reviewer: I tried
